bin/telemetry_sim_ATHS

```diff
@@ -110,7 +117,14 @@ if __name__ == "__main__":
             'priority': 1,
             'force_output': True}
     ptg.evt_currentTarget.set_put(**kwTA)
-    LOGGER.info("Sending currentTarget")
+    LOGGER.info("Sending Ptg currentTarget")
```

Comment:
What do you mean by "sending" in this context?? It might be better to use the word "Sending simulated Ptg currentTarget event"

(also in line 111 "Poiting" is missing an 'n' :)

Reply:
This also applies to all the "sending" strings below as well.

Reply:
Yes, these are all simulated messages. I'll add to the logger.